### DISCUSSION TITLE
Example of Ollama with OpenAI proxy server

### DISCUSSION THREAD

**Discussion by jxnl** [BerriAI/litellm#1336] [Q&A]
Is there an example of using ollama with the openai proxy server to host mistral?

**Comment by ghost** [comment]
Yep right here - https://docs.litellm.ai/docs/proxy/quick_start#supported-llms

```
$ litellm --model ollama/<ollama-model-name>
```

e.g. 

```
$ litellm --model ollama/codellama
```

to call ollama's codellama model (by default this will assume it's on port 11434)

If you want to change the api base, just do 

```
$ litellm --model ollama/<ollama-model-name> --api_base <my-hosted-endpoint>
```

**Comment by ghost** [comment]
@jxnl let me know if this solves your problem
